### DISCUSSION TITLE
Cannot send mail

### DISCUSSION THREAD

**Discussion by ghost** [saran-surya/email_auth#44] [Q&A]
Hello, i'm a newbie. i've followed all your steps but when i hit send button, there was a message that "Unable to send mail" although the remote configuration are valid. Please help me with that.

**Comment by saran-surya** [comment]
Hi @duyhuynh-arc97 Kindly post the flutter logs, It will be helpful to understand the problem better

**Comment by saran-surya** [comment]
Hi @duyhuynh-arc97 , Is your issue resolved ?
